spread.yaml

```diff
@@ -68,22 +68,22 @@ prepare: |
     sysctl -w net.ipv6.conf.all.disable_ipv6=1
     trap "sysctl -w net.ipv6.conf.all.disable_ipv6=0" EXIT
 
-    apt purge -y snapd || true
-    apt update
+    apt-get purge -y snapd || true
+    apt-get update
     # utilities
-    apt install -y devscripts expect jq rng-tools software-properties-common
+    apt-get install -y devscripts expect jq rng-tools software-properties-common
 
     # needed so that we have golang-gopkg-macaroon.v1 which is not (yet)
     # in trusty
     add-apt-repository --update ppa:snappy-dev/image
     # this should not be needed but apparently it is :/
-    apt update
+    apt-get update
 
-    apt build-dep -y ./
+    apt-get install -y $(gdebi --quiet --apt-line ./debian/control)
```

Comment:
if I understand @mvo5 was proposing to leave the future way at least as a comment

# in 16.04: apt build-dep -y ./